Sources/Subs.php

```diff
@@ -7393,6 +7393,15 @@ function validate_iri($iri, $flags = null)
 {
 	$url = iri_to_url($iri);
 
+	// PHP 5 doesn't recognize IPv6 addresses in the URL host.
+	if (version_compare(phpversion(), '7.0.0', '<'))
+	{
+		$host = parse_url((strpos($url, '//') === 0 ? 'http:' : '') . $url, PHP_URL_HOST);
+
+		if (strpos($host, '[') === 0 && strpos($host, ']') === strlen($host) && strpos($host, ':') !== false)
```

Comment:
Cool workaround but should be strpos($host, ']') === strlen($host) **- 1** to work

Reply:
Derp.